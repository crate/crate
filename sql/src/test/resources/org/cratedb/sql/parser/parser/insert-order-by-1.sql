insert into t (x, y, z) select a, b, c from t2 order by a