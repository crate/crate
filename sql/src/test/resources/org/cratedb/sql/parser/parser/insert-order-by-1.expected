org.cratedb.sql.parser.parser.InsertNode@2345f0e3
statementType: INSERT
resultSet: 	
	org.cratedb.sql.parser.parser.SelectNode@44c9d92c
	isDistinct: false
	resultColumns: 		
		org.cratedb.sql.parser.parser.ResultColumnList@1fd0fafc

		[0]:		
		org.cratedb.sql.parser.parser.ResultColumn@510dc6b5
		exposedName: a
		name: a
		tableName: null
		isDefaultColumn: false
		type: null
		expression: 			
			org.cratedb.sql.parser.parser.ColumnReference@5f70bea5
			columnName: a
			tableName: null
			type: null
		[1]:		
		org.cratedb.sql.parser.parser.ResultColumn@62f47396
		exposedName: b
		name: b
		tableName: null
		isDefaultColumn: false
		type: null
		expression: 			
			org.cratedb.sql.parser.parser.ColumnReference@1ed0af9b
			columnName: b
			tableName: null
			type: null
		[2]:		
		org.cratedb.sql.parser.parser.ResultColumn@16b8f8eb
		exposedName: c
		name: c
		tableName: null
		isDefaultColumn: false
		type: null
		expression: 			
			org.cratedb.sql.parser.parser.ColumnReference@49de17f4
			columnName: c
			tableName: null
			type: null
	fromList: 		
		org.cratedb.sql.parser.parser.FromList@13f6ba0f

		[0]:		
		org.cratedb.sql.parser.parser.FromBaseTable@2b313906
		tableName: t2
		updateOrDelete: null
		null
		correlation Name: null
		null
targetTableName: 	
	org.cratedb.sql.parser.parser.TableName@74
	t
targetColumnList: 	
	org.cratedb.sql.parser.parser.ResultColumnList@2c96cf11

	[0]:	
	org.cratedb.sql.parser.parser.ResultColumn@60f47bf5
	exposedName: x
	name: x
	tableName: null
	isDefaultColumn: false
	type: null
	reference: 		
		org.cratedb.sql.parser.parser.ColumnReference@52f6438d
		columnName: x
		tableName: null
		type: null
	[1]:	
	org.cratedb.sql.parser.parser.ResultColumn@25cd0888
	exposedName: y
	name: y
	tableName: null
	isDefaultColumn: false
	type: null
	reference: 		
		org.cratedb.sql.parser.parser.ColumnReference@37eb2c1b
		columnName: y
		tableName: null
		type: null
	[2]:	
	org.cratedb.sql.parser.parser.ResultColumn@309cbd4b
	exposedName: z
	name: z
	tableName: null
	isDefaultColumn: false
	type: null
	reference: 		
		org.cratedb.sql.parser.parser.ColumnReference@24f9fdc
		columnName: z
		tableName: null
		type: null
orderByList: 	
	org.cratedb.sql.parser.parser.OrderByList@398825b3
	allAscending: true
	[0]:	
	org.cratedb.sql.parser.parser.OrderByColumn@19632847
	columnPosition: -1
	expression: 		
		org.cratedb.sql.parser.parser.ColumnReference@3ce95a56
		columnName: a
		tableName: null
		type: null